pytest-selenium/pages/base.py

```diff
@@ -75,3 +75,15 @@ def height(self, element):
                 "return window.getComputedStyle(arguments[0]).height;", element
             )
         ).strip("px")
+
+    def offscreen_click(self, element=None):
+        """Clicks an offscreen element (or the region's root).
+
+        Clicks the given element, even if it is offscreen, by sending the ENTER key.
+        Returns the element.
+        """
+        # We actually navigate using the ENTER key because scrolling the page can be flaky
+        # https://stackoverflow.com/a/39918249
+        # return self.page.offscreen_click(element or self.root)
+        element.send_keys(Keys.ENTER)
```

Comment:
This may raise an exception if the `element=None` or the `find_element` in use by `element` fails; is this handled elsewhere or is a raised exception acceptable when running the test (ie: not in an `Exception` as opposed to an assertion statement)?

Reply:
I think its ok as long the test is failing when it should. We haven't been catching exceptions in this case.